src/components/VColorPaletteMaterial.vue

```diff
@@ -63,6 +67,9 @@ export default {
     value() {
       this.setColorFromInput();
     },
+    hexColor() {
```

Comment:
You can achieve same functionality without watcher, with computed property. Set computed value hexNum as: _return this.hexColor ? this.hexColor.value : null;_